uat/mqtt-client-control/src/main/java/com/aws/greengrass/testing/mqtt/client/control/api/EventFilter.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.testing.mqtt.client.control.api;
+
+import lombok.NonNull;
+
+import java.nio.charset.StandardCharsets;
+
+/**
+ * Filter of events in event store.
+ */
+public final class EventFilter {
+    private final Event.Type type;
+    private final String connectionName;
+    private final String topic;
+    private final String topicFilter;
+    private final byte[] content;
+
+    EventFilter(Builder builder) {
+        super();
+        this.type = builder.type;
+        this.connectionName = builder.connectionName;
+        this.topic = builder.topic;
+        this.topicFilter = builder.topicFilter;
+        this.content = builder.content;
+    }
+
+    /**
+     * Builder of event filter.
+     */
+    public static class Builder {
```

Comment:
we can use Lombok to get rid of all the boilerplate code

Reply:
We have overloaded withContent() with String and byte[] with lombok we will have only one withContent(byte[])